python/llm/src/bigdl/llm/langchain/embeddings/bigdlllm.py

```diff
@@ -190,3 +200,191 @@ def embed_query(self, text: str) -> List[float]:
         """
         embedding = self.client.embed(text)
         return list(map(float, embedding))
+
+
+class _BaseEmbeddings(BaseModel, Embeddings):
```

Comment:
Can BigdlNativeEmbeddings merge the code with this class? Currently too much duplicate code

Reply:
I don't think so :( .